core/controllers/release_coordinator.py

```diff
@@ -49,3 +54,121 @@ def get(self) -> None:
     def delete(self) -> None:
         caching_services.flush_memory_caches()
         self.render_json({})
+
+
+class FeatureFlagsNormalizePayloadDict(TypedDict):
```

Comment:
Normalize --> Normalized?

Also the name here doesn't make sense, a normalized payload dict should correspond to a handler, not "feature flags". Feature flags don't have an action, for example.